listenbrainz_spark/utils.py

```diff
@@ -8,3 +16,34 @@ def create_path(path):
     except OSError as e:
         if e.errno != errno.EEXIST:
             raise
+
+
+def create_app():
+    app = CustomFlask(import_name=__name__)
+    config_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'config.py')
+    for i in range(APP_CREATION_RETRY_COUNT):
```

Comment:
If this loop is exited it will likely die with a stacktrace later, which isn't so nice. And else clause on the for loop could take care of printing a message and.. exiting?